### ISSUE TITLE
'--export-pdf' argument not accepted

### ISSUE THREAD

**Issue by mgpinf** [mfontanini/presenterm#59]
I installed `presenterm-export` and `tmux` and upon running the command to export to pdf, the below message is displayed:
```
[~/Downloads/presenterm-0.2.1]$ ./presenterm  --export-pdf ../hw1.md
error: unexpected argument '--export-pdf' found

  tip: to pass '--export-pdf' as a value, use '-- --export-pdf'

Usage: presenterm [OPTIONS] <PATH>

For more information, try '--help'.
```

**Comment by mfontanini** [comment]
This change is not released yet. You can download the latest source and run it from there. I'm preparing the next release and I'm hoping I can get that done later today.

**Comment by mfontanini** [comment]
I just released version 0.3.0. Also I'd suggest updating to presenterm-export 0.1.4 as I fixed an issue with relative paths.
